components/Banner/README.md

```diff
@@ -1,8 +1,156 @@
 
 A banner displays a prominent message and related optional actions.
-Note: this component is currently being actively developed and is not yet ready for general use.
+
+<div class="article__asset article__asset--screenshot">
+  <img src="docs/assets/banner.png" alt="Banner" width="320">
+</div>
+
+## Design & API documentation
+
+
+## Table of contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+  - [Installation with CocoaPods](#installation-with-cocoapods)
+  - [Importing](#importing)
+- [Usage](#usage)
+  - [Importing](#importing)
+  - [Appearance](#appearance)
+  - [Styling](#styling)
+  - [LayoutMargins](#layoutmargins)
+- [Examples](#examples)
+  - [Creating a banner view](#creating-a-banner-view)
+
+- - -
+
+## Overview
+
+`MDCBannerView` is a view that displays important, succinct message, and provides actions for users to address (or dismiss the banner). It requires a user action to be dismissed.
+
+`MDCBannerView` is currently a [beta component](../../contributing/beta_components.md). Therefore, clients that
+wish to use MDCBannerView in their app will need to manually specify the MaterialComponentsBeta target and our repository. 
+
+## Installation
+
+<!-- Extracted from docs/../../../docs/component-installation.md -->
+
+### Installation with CocoaPods
+
+Add the following to your `Podfile`:
+
+```bash
+pod 'MaterialComponents/Banner'
+```
+<!--{: .code-renderer.code-renderer--install }-->
+
+Then, run the following command:
+
+```bash
+pod install
+```
+
+### Importing
+
+To import the component:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+import MaterialComponents.MaterialBanner
+```
+
+#### Objective-C
+
+```objc
+#import "MaterialBanner.h"
+```
+<!--</div>-->
+
+
+## Usage
+
+<!-- Extracted from docs/typical-use.md -->
+
+### Importing
+
+To use the `MDCBannerView` in your code, import the MaterialBanner umbrella header (Objective-C) or MaterialComponents module (Swift).
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+
+```swift
+import MaterialComponents
+```
+
+#### Objective-C
+
+```objc
+#import "MaterialBanner.h"
+```
+
+<!--</div>-->
+
+### Appearance
+
+By default, the `MDCBannerView` is configured to display an image view, a text label and two buttons. To hide the image view on `MDCBannerView`, users can set the `hidden` property on `imageView` to be true. Similarly, users can hide a button on banner view by setting the `hidden` property on `trailingButton` to be true.
+
+### Styling
+
+By default, the `MDCBannerView` is configured to display items with black text and white background. To customize the color and style of the text, image view and buttons displayed on `MDCBannerView`, directly set relevant properties, such as `tintColor`, on `textLabel`, `imageView`, `leadingButton` and `trailingButton`.
```

Comment:
nit: directly set the relevant properties

Reply:
nit: remove "on" from `textLabel`

Reply:
"on" is needed since tintColor will be set on `textLabel`.